src/main/webapp/app/tutor-exercise-dashboard/tutor-exercise-dashboard.component.ts

```diff
@@ -293,6 +294,7 @@ export class TutorExerciseDashboardComponent implements OnInit {
             route = `/modeling-exercise/${this.exercise.id}/submissions/${submission}/assessment`;
             queryParams.showBackButton = true;
         }
+        // TODO: Make sure route is set
```

Comment:
@SiggZ Can you make sure `route` is set?

Reply:
I added a fix in ce859b5. Can you verify that it is ok?

Reply:
The fix looks good to me.